404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
   <li>Worldwide payments</li>
-  <li>Low or zero processing fees</li>
```

Comment:
>Low or zero processing fees

This is somewhat misleading, but there's a core of truth in it. I suggest changing it to:

>Low negational processing fees.

Because up to the latest block, fees are *low* (cents) but not "Free". Also, it's market driven.